lib/WebpackConfig.js

```diff
@@ -173,6 +173,11 @@ class WebpackConfig {
         this.terserPluginOptionsCallback = () => {};
         this.optimizeCssPluginOptionsCallback = () => {};
         this.notifierPluginOptionsCallback = () => {};
+        this.useImagesLoader = true;
+        this.useFontsLoader = true;
```

Comment:
This line and the one above it already exist at the beginning of the `constructor`.

More generally, the settings have been divided into categories (flags, options, callback), so you'll also have to move `useBrowserSync`, `browserSyncOptionsCallback` and `browserSyncOptions` a bit.